test/runtests.jl

```diff
@@ -2,6 +2,52 @@ using Zygote, Test
 using Zygote: gradient
 using CUDA: has_cuda
 
+using Zygote
+@testset "gradient checkpointing" begin
```

Comment:
In which file should these tests live? They are here right now only for quicker development iteration.

Reply:
@DhairyaLGandhi ?

Reply:
This would be good for `features.jl`